src/eval-frame/components/reps/__test_helpers__/reps-test-value-cases.js

```diff
@@ -0,0 +1,283 @@
+export const simpleTypes = {
+  number_int: 232937,
+  number_float: 453.12312312,
+  number_zero: 0,
+  number_negative: -534.23,
+  number_exp: 10e55,
+  number_nan: NaN,
+  number_inf: Infinity,
+  string_empty: "",
+  string_short: "asjhdflkdskfhla",
+  boolean_true: true,
+  boolean_false: false,
+  simple_undef: undefined,
+  simple_null: null,
+  symbol: Symbol.for("test_symbol"),
+  symbol_long: Symbol.for("test_symbol_with_a_very_long_symbol_name")
+};
+
+// ==================== longStrings
+
+export const longStrings = {
+  string_withSpaces_100: "1234 ".repeat(20),
+  string_withSpaces_1000: "1234 ".repeat(200),
+  string_withSpaces_100000: "1234 ".repeat(20000),
+  string_noSpaces_100: "12345".repeat(20),
+  string_noSpaces_1000: "12345".repeat(200),
+  string_noSpaces_100000: "12345".repeat(20000)
+};
+
+// ==================== arrays
+
+const N_ARRAY = 10000;
+
+export const longValueArrays = {};
+Object.keys(simpleTypes).forEach(type => {
+  longValueArrays[`array_long_${type}`] = new Array(N_ARRAY).fill(
+    simpleTypes[type]
+  );
+});
+longValueArrays.array_long_mixed = new Array(1000)
+  .fill(Object.values(simpleTypes))
+  .reduce((a, b) => a.concat(b), []);
+
+export const arrayBuffers = {
+  buffer_arrayBuffer: new ArrayBuffer(N_ARRAY),
```

Comment:
what about `Blob`? 

(sidenote - remember the movie ["What About Bob?"](https://en.wikipedia.org/wiki/What_About_Bob%3F)? did that one age well?)

Reply:
thanks, added.

let's just keep adding more things as we think of them in the future. (at some point soon we'll need to add BigInt for example. Chrome already has it, but I don't think we need to add it yet)